tasks_downstream.py

```diff
@@ -1,11 +1,12 @@
 """Doodba child project tasks.
 
-This file is to be executed with https://www.pyinvoke.org/ in Python 3.6+.
+This file is to be executed with https://www.pyinvoke.org/ in Python 3.8.1+.
```

Comment:
Is a requirement P3.8? There's still Doodba versions with lower Python versions?

Reply:
One thing ar dev tools (such as invoke), and other thing is Odoo (which runs inside the container, with a specified python version). Py3.6 is EOL since long ago.

Reply:
OK, got it.